tests/core/test_evaluation.py

```diff
@@ -91,3 +92,22 @@ async def test_end_to_evaluation_with_forms(form_bot_agent: Agent):
     )
 
     assert not story_evaluation.evaluation_store.has_prediction_target_mismatch()
+
+
+async def test_source_in_failed_stories(tmpdir: Path, default_agent: Agent):
+    stories_path = str(tmpdir / "failed_stories.md")
```

Comment:
Thanks a lot for adding the tests! 👍 

One more thing - could you please extract the `failed_stories.md` as a constant in `rasa/core/test.py` and reuse it here? Just to be future proof.

Reply:
Done